cdap-distributions/src/etc/cdap/conf.dist/logback-container.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  Copyright © 2015 Cask Data, Inc.
+    
+  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+  use this file except in compliance with the License. You may obtain a copy of
+  the License at
+    
+  http://www.apache.org/licenses/LICENSE-2.0
+    
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  License for the specific language governing permissions and limitations under
+  the License.
+  -->
+
+<configuration>
+
+  <!--
+    Disabling some chatty loggers.
+  -->
+  <logger name="org.apache.commons.beanutils" level="ERROR"/>
+  <logger name="org.apache.zookeeper.server" level="ERROR"/>
+  <logger name="org.apache.zookeeper" level="ERROR"/>
+  <logger name="com.ning" level="WARN"/>
+  <logger name="org.apache.hadoop" level="WARN"/>
+  <logger name="org.apache.hive" level="WARN"/>
+  <logger name="org.quartz.core" level="WARN"/>
+  <logger name="co.cask.cdap" level="INFO"/>
+  <logger name="Explore.stdout" level="INFO"/>
+  <logger name="Explore.stderr" level="INFO"/>
+
+    <appender name="Rolling" class="ch.qos.logback.core.rolling.RollingFileAppender">
+      <file>${LOG_DIRS}/program.log</file>
+      <encoder>
+        <pattern>%d{ISO8601} - %-5p [%t:%logger{1}@%L] - %m%n</pattern>
+      </encoder>
+      <rollingPolicy class="ch.qos.logback.core.rolling.FixedWindowRollingPolicy">
+        <maxIndex>10</maxIndex>
+        <FileNamePattern>${LOG_DIRS}/program.log.%i</FileNamePattern>
+      </rollingPolicy>
+      <triggeringPolicy class="ch.qos.logback.core.rolling.SizeBasedTriggeringPolicy">
+        <MaxFileSize>1MB</MaxFileSize>
```

Comment:
It might be good to use `TimeBasedRollingPolicy` and rollover log files every day, and keep a history of 10.